src/components/sbb-option/sbb-option.tsx

```diff
@@ -149,6 +148,11 @@ export class SbbOption implements ComponentInterface {
   @Method()
   public async setSelectedViaUserInteraction(selected: boolean): Promise<void> {
     this.selected = selected;
+    this.selectionChange.emit({
```

Comment:
Do we really need the payload from this event?
Because id, value and selected state could simply be read from the event `target`. Or am I misunderstanding the purpose?

Reply:
Great point, removed in [44c81bb](https://github.com/lyne-design-system/lyne-components/pull/1843/commits/44c81bb2443969c3ebc7ae3b341f2cde525a43f2) 👍